db/models.py

```diff
@@ -14,3 +14,43 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    actors = models.ManyToManyField(Actor, related_name="actor")
+    genres = models.ManyToManyField(Genre, related_name="genre")
```

Comment:
The `related_name` for the `genres` field in the `Movie` model is set to 'genre'. Similar to the `actors` field, using a plural form like 'movies' would be more appropriate to indicate the reverse relation from `Genre` to `Movie`.